forms-flow-data-analysis-api/src/api/resources/sentiment_analysis.py

```diff
@@ -5,16 +5,15 @@
 from flask import jsonify, request
 from flask_restx import Namespace, Resource, cors
 
-from api.services.sentiment_analysis import sentiment_analysis_pipeline
 from api.services.store_sentiment_result import save_sentiment_result
+from api.services.transformers import sentiment_analysis_pipeline_transformers
 from api.utils import auth, cors_preflight
 
 API = Namespace("sentiment", description="API endpoint for sentiment analysis")
 
-
 @cors_preflight("POST,OPTIONS")
-@API.route("", methods=["POST", "OPTIONS"])
-class SentimentAnalysisResource(Resource):
+@API.route("/predict", methods=["POST", "OPTIONS"])

```

Comment:
any reason to change the endpoint name ?

Reply:
This was done before removing the old endpoint, will update the name to old one